package-lock.json

```diff
@@ -1,91 +1,15130 @@
 {
   "name": "nextcloud-mail",
   "version": "1.7.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
what version of npm do you use?

Reply:
7.0.9 (Debian bullseye)

Reply:
Did you install a custom version? https://blog.npmjs.org/post/631877012766785536/release-v700 it's been release a month ago but my node v15.1.0 still comes with npm 6.14.8.

What's the output for
```
node --version
which npm
```
?

We'll have to migrate the lockfile at some point but I'm curious why you have the new npm already.

Reply:
cyrille@vm-cyrille-debian:/var/www/nextcloud/apps/mail$ npm --version
7.0.9
cyrille@vm-cyrille-debian:/var/www/nextcloud/apps/mail$ which npm
/usr/bin/npm
cyrille@vm-cyrille-debian:/var/www/nextcloud/apps/mail$ dpkg -S /usr/bin/npm
npm: /usr/bin/npm
cyrille@vm-cyrille-debian:/var/www/nextcloud/apps/mail$ apt show npm
Package: npm
Version: 7.0.11+ds-1
...